pkg/util/crypt/crypt_dev.go

```diff
@@ -71,29 +70,22 @@ func (crypt *Device) CloseCryptDevice(path string) error {
 // Keys should be non-interactive, preferably in a keyfile that can
 // be passed to cryptsetup utility
 func (crypt *Device) ReadKeyFromStdin(confirm bool) (string, error) {
-
-	fmt.Print("Enter the Key: ")
-	password, err := terminal.ReadPassword(int(syscall.Stdin))
+	pass1, err := sypgp.AskQuestionNoEcho("Enter a passphrase: ")
 	if err != nil {
-		sylog.Fatalf("Error parsing the key: %s", err)
+		return "", fmt.Errorf("unable getting input: %s", err)
```

Comment:
Avoid using general words like "input" on error messages .. maybe something better would be: "unable to get passphrase"